pkgs/games/starruler2/default.nix

```diff
@@ -0,0 +1,59 @@
+{
+  lib,
+  stdenv,
+  callPackage,
+  makeWrapper,
+  # Path to directory containing music for the game,
+  # such as `data/music` from the commercial version.
+  musicPath ? null,
+  # Paths of mods to add.
+  modPaths ? [],
+}: let
+  data = callPackage ./data.nix {};
+  engine = callPackage ./engine.nix {};
+in
+  # The game cannot access files outside the working directory,
+  # so it cannot access files behind symlinks.
```

Comment:
As I said in my comment, would be better to say why "The game cannot access files outside the working directory"

Reply:
I rewrote the comment. Let me know if it is clear enough now.

Reply:
Perfect !